content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,109 @@
+
+### Tags and authors pages
+
+We now have a [tags page](/tags) referencing all tags along with the number of posts associated to them. The tags of a blog post, tech note or comm call are shown at the bottom, so that after reading you can choose to check out more content on the topic. After tags became even more valuable, our community manager Stefanie Butland added many of them and for making that process smoother we've created a [Shiny app to update tags more easily over many posts, `blogyaml`](https://github.com/ropenscilabs/blogyaml). It's worked well for us, and if you notice something wrong, well it has an issue tracker!
+
+We also have added an [authors page](/authors) that's very similar. On each author's page, you'll find their title for rOpenSci staff members, their online home, possibly their Twitter handle, and all the content they've authored on our website. [See for instance Stefanie Butland's author page](authors/stefanie-butland). These pages are linked from the bottom of posts for returning authors. Thanks to [Angela Li](https://angela-li.github.io/) for [requesting we added authors pages](https://github.com/ropensci/roweb2/issues/365).
+
+Technically, getting tags and authors page and tag pages is quite straightforward with Hugo thanks to _taxonomies_. To switch them on, we added these lines to the website config.toml
+
+```
+[taxonomies]
+  tag = "tags"
+  author = "authors"
+```
+
+We added this option as well
+
+```
+preserveTaxonomyNames = true
+```
+
+to not mess up with authors' names capitalization when listing them.
+
+For authors, a challenge was moving metadata around, from storing authors' metadata (online home, Twitter username) inside posts metadata to having authors metadata files under content/authors, see [this example](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md), but as mentioned earlier, with the `yaml` package you can edit hundreds of Markdown files at once, granted you check the edits carefully.
+
+Read more about [Hugo taxonomies](https://gohugo.io/content-management/taxonomies/), how to write [templates for taxonomy pages](https://gohugo.io/templates/taxonomy-templates/) and [where to save them so Hugo finds them](https://gohugo.io/templates/lookup-order/#examples-layout-lookup-for-taxonomy-list-pages). In our case, the tags and authors pages have different templates: [tags page](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/terms.html), [tag pages](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/taxonomy/tag.html), [authors templates](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts/authors)
```

Comment:
might need a few more explanatory words since a reader won't know difference between "tags page" and "tag pages" without clicking. And with so many links they might not click